lib/vc.js

```diff
@@ -162,7 +310,92 @@ function _getId(obj) {
   return obj.id;
 }
 
-async function _checkCredential({credential}) {
+/**
+ * Checks if a given credential is a VerifiablePresentation.
+ *
+ * @param credential
+ * @returns {boolean}
+ */
+function _isPresentation(credential) {
+  return credential['type'] &&
+    jsonld.getValues(credential, 'type').includes('VerifiablePresentation');
+}
+
+/**
+ * @param presentation
+ * @throws {Error}
+ * @private
+ */
+function _checkPresentation(presentation) {
+  if(!presentation['@context']) {
+    throw new Error('"context" property is required.')
+  }
+
+  // check context cardinality
+  if(jsonld.getValues(presentation, '@context').length <= 1) {
+    throw new Error('"context" property needs to be an array of two or more contexts.')
+  }
+
+  // ensure first context is 'https://www.w3.org/2018/credentials/v1'
+  if(presentation['@context'][0] !== 'https://www.w3.org/2018/credentials/v1') {
+    throw new Error('credentials/v1 context needs to be first in the list of contexts.')
```

Comment:
Here and similar cred error should show full URL: `https://www.w3.org/2018/credentials/v1`

Reply:
Fixed, thanks.